packages/react/runtime/compat/index.js

```diff
@@ -0,0 +1,16 @@
+// Copyright 2025 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+import { startTransition, useTransition } from 'preact/compat';
+
+/* eslint-disable-next-line import/default */
+import ReactLynx from '@lynx-js/react';
+
+export default {
+  ...ReactLynx,
+  startTransition,
+  useTransition,
+};
```

Comment:
The `...ReactLynx` may have side effect, which cause this module not tree shakeable. Maybe use `#__PURE__` with `Object.assign()` instead.


```suggestion
export default /*#__PURE__*/ Object.assign({}, ReactLynx, {
  startTransition,
  useTransition,
});
```